<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwidget</class>
 <widget class="QMainWindow" name="mainwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: silver;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>531</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: silver;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Load</string>
     </attribute>
     <widget class="QPushButton" name="loadButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#loaButton {
background: grey; 
color: pink;
 font-family: Menlo;
 border: 1px solid black; 
border-radius: 8px;
}
#loadButton:hover {
	color: green;
	font-size: 14px;
	background: white;
}
#loadButton:pressed {
	font-size: 15px;
	color: darkgreen;
	background: darkgrey;
}</string>
      </property>
      <property name="text">
       <string>Load model</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Move</string>
     </attribute>
     <widget class="QPushButton" name="moveButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#moveButton {
background: silver; 
color: black;
 font-family: Menlo;
 border: 1px solid black; 
border-radius: 3px;
}
#moveButton:hover {
	color: green;
	font-size: 14px;
	background: white;
}
#moveButton:pressed {
	font-size: 15px;
	color: darkgreen;
	background: darkgrey;
}</string>
      </property>
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dx_input">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dy_input">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dz_input">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>dX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>dY</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>dZ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Rotate</string>
     </attribute>
     <widget class="QPushButton" name="rotateButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#rotateButton {
background: silver; 
color: black;
 font-family: Menlo;
 border: 1px solid black; 
border-radius: 3px;
}
#rotateButton:hover {
	color: green;
	font-size: 14px;
	background: white;
}
#rotateButton:pressed {
	font-size: 15px;
	color: darkgreen;
	background: darkgrey;
}</string>
      </property>
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
     <widget class="QLabel" name="z_angle">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>dZ</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ry_input">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rx_input">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rz_input">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="x_angle">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>dX</string>
      </property>
     </widget>
     <widget class="QLabel" name="y_angle">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>dY</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Scale</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>Scale C.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="scaleButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#scaleButton {
background: silver; 
color: black;
 font-family: Menlo;
 border: 1px solid black; 
border-radius: 3px;
}
#scaleButton:hover {
	color: green;
	font-size: 14px;
	background: white;
}
#scaleButton:pressed {
	font-size: 15px;
	color: darkgreen;
	background: darkgrey;
}</string>
      </property>
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scale_input">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: white;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
